aptos-move/move-examples/mint_nft_v2/1-Create-NFT/sources/create_nft.move

```diff
@@ -0,0 +1,187 @@
+/// This is a tutorial intended to mirror the `mint_nft` example in move-examples as closely as possible, with the only difference
+/// being that we're using the no code token from aptos_token to demonstrate how to do the tutorial with token v2 instead of token v1.
```

Comment:
For after you turn this into a tutorial/doc on aptos.dev, we should just include the link to @gregnazario's doc on Token v2 which should already compare token v1 and v2. This tutorial shouldn't mention much about token v1 at all as it could be confusing to new devs who only see token v2 (which is the recommended approach now) and don't need to worry about v1.